src/pages/user-list/user-list.tsx

```diff
@@ -22,15 +23,17 @@ export const UserList = () => {
   const [searchValue, setSearchValue] = useState('');
 
   const getUsersData = async () => {
-    fetch('https://dummyapi.io/data/v1/user?limit=20', {
-      headers: {
-        'app-id': '645945fa70a1298f9606d753',
-      },
-    }).then((response) => response.json())
-      .then((response) => {
-        setUsers(response.data);
-        setIsLoading(false);
+    try {
+      const response = await axios.get('https://dummyapi.io/data/v1/user', {
+        headers: {
+          'app-id': '645945fa70a1298f9606d753',
+        },
       });
+      setIsLoading(false);
+      setUsers(response.data.data);
+    } catch {
+      console.log('Error');
```

Comment:
```suggestion
      setIsLoading(false);
      console.log('Error');
```
we have to add it because if the execution of de get has an error, the state of isLoading is still in true

Reply:
maybe you can change the error to show the same error as in the sign-in and login